### PR TITLE
Yarn on-the-metal

### PR DESCRIPTION
**Pull request by jennydaman**
- references to `npm` replaced with `yarn` in `package.json`
- `yarn serve` replaced with `sirv`, much smaller and more efficient than the express.js-powered `serve.js`
- patch version bumped `0.1.0` --> `0.1.1`

### CODE REVIEW

**Comment by anujsingla** [comment]
@jennydaman Why you have deleted server.js file, any reason?

**Comment by jennydaman** [comment]
@anujsi Just trying to minimize boilerplate and `node_modules`

https://packagephobia.com/result?p=express (1.61 MB)
https://packagephobia.com/result?p=sirv-cli (189 kB)

**Comment by anujsingla** [comment]
@jennydaman We are using server.js file in the package.json file, 
https://github.com/FNNDSC/ChRIS_store_ui/blob/master/package.json#L12
I think we should not delete this file.

File: package.json [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "lint": "eslint ./src/components",
     "lint:fix": "eslint --fix ./src/components",
-    "ci": "concurrently --kill-others-on-fail \"npm:test\" \"npm:lint\"",
-    "serve": "node serve.js",
-    "deploy": "npm run build && npm run serve",
+    "ci": "concurrently --kill-others-on-fail \"yarn:test\" \"yarn:lint\"",
+    "serve": "sirv build --cors --single --host --port 3000",
```
Comment (jennydaman) [inline package.json:12]: @anujsingla *were using `server.js` but the objective of this PR is to _replace_ `serve.js`

**Comment by anujsingla** [comment]
@jennydaman Reviewed, Please test it properly.

**Comment by jennydaman** [comment]
Github requires that you click the "Review changes" button to submit a review.

I have tested it.

**Review by anujsingla — APPROVED** [review]
Reviewed.

**Pull request merged**